src/init/features/dataconnect/index.ts

```diff
@@ -65,17 +66,27 @@ const defaultConnector = {
 
 // doSetup is split into 2 phases - ask questions and then actuate files and API calls based on those answers.
 export async function doSetup(setup: Setup, config: Config): Promise<void> {
-  const info = await askQuestions(setup, config);
+  const info = await askQuestions(setup);
   await actuate(setup, config, info);
+
+  const promptForSDKGeneration = await confirm({
+    message: `Would you like to configure generated SDKs now?`,
+    default: true,
```

Comment:
Default to true or false?

I think we can run a quick framework detection, if something is detected, go straight to SDK setup w/o prompting.
If nothing is detected, then ask if they want to setup SDK and default to false.

https://github.com/firebase/firebase-tools/blob/master/src/init/features/dataconnect/sdk.ts#L70-L71

Reply:
Existing code, but I think "which connector" question should be asked after picking the framework.

https://github.com/firebase/firebase-tools/blob/master/src/init/features/dataconnect/sdk.ts#L39-L65

Reply:
Updated to run framework detection and proceed with gen SDK setup w/o prompting when a framework is detected.

I don't think we can simplify by including "skip" as an option in the framework question, as if a framework isn't detected we prompt for an app directory first before asking the framework question, which results in weird UX if we make them enter an app directory before they can skip out of the gen SDK flow.

Reply:
Works for me. Thanks!